pkg/plugins/discovery/aws/provider.go

```diff
@@ -70,6 +70,7 @@ func (p *eksClusterProvider) ConfigurationItems() config.ConfigurationSet {
 	cs := config.NewConfigurationSet()
 	cs.String("partition", endpoints.AwsPartition().ID(), "AWS partition to use") //nolint: errcheck
 	cs.String("region", "", "AWS region to connect to")                           //nolint: errcheck
+	cs.String("region-filter", "", "A filter to apply to the AWS regions list")   //nolint: errcheck
```

Comment:
Would it be worth adding an example to the description? "A filter to apply to the AWS regions list. If you specify 'us-' it will filter the regions listed to only those in the US"

Reply:
Agreed